src/ontology/mondo-edit.obo

```diff
@@ -453470,9 +453470,9 @@ id: MONDO:0100349
 name: COACH syndrome
 def: "A Mendelian disease characterized by infantile ataxia with hypo/aplastic vermis, hepatic fibrocirrhosis, slender-shaped skeleton, peculiar face, and moderate intellectual disability." [PMID:2929661]
 synonym: "cerebellar vermis hypoplasia, oligophrenia, ataxia, colobomas, and hepatic fibrosis" EXACT [OMIM:619111]
-xref: OMIM:216360
-xref: OMIM:619111
-xref: OMIM:619113
+xref: OMIM:216360 {source="MONDO:mondoIsBroaderThanSource"}
```

Comment:
We haven't been adding these consistently. I don't think it's an issue to add this but I added this to the QC agenda because I'm not sure what our plan is with this.